src/app/config/schema.ts

```diff
@@ -303,6 +303,30 @@ export const optionalVarsSchema: Schema<IOptionalVarsSchema> = {
       default: '',
       env: 'VIRUS_SCANNER_LAMBDA_ENDPOINT',
     },
+    guarddutyQuarantineS3Bucket: {
+      doc: 'New guardduty S3 Bucket to quarantine files for virus scanning',
+      format: String,
+      default: '',
+      env: 'GUARDDUTY_QUARANTINE_S3_BUCKET',
+    },
+    guarddutyCleanS3Bucket: {
+      doc: 'New S3 Bucket to store files that have been scanned and are clean from guardduty quarantine bucket',
+      format: String,
+      default: '',
+      env: 'GUARDDUTY_CLEAN_S3_BUCKET',
+    },
+    guarddutyLambdaEndpoint: {
+      doc: 'Endpoint address for virus scanner lambda function. Specify this if the lambda is hosted neither on AWS nor your local dev environment.',
+      format: String,
+      default: '',
+      env: 'GUARDDUTY_LAMBDA_ENDPOINT',
+    },
+    guarddutyLambdaFunctionName: {
+      doc: 'Virus scanner lambda function name',
```

Comment:
Best that we retain the documentation to note that this envvar is optional.
```suggestion
      doc: 'Endpoint address for guardduty virus scanner lambda function. Specify this if the lambda is hosted neither on AWS nor your local dev environment',
```

Reply:
Done!